include/OpenSpaceToolkit/Astrodynamics/Trajectory/StateBuilder.hpp

```diff
@@ -91,7 +117,22 @@ class StateBuilder
     ///
     /// @return                 A State linked to the Frame and Coordinates Broker of the StateBuilder
 
-    const State buildState(const Instant& anInstant, const VectorXd& aCoordinates) const;
+    const State build(const Instant& anInstant, const VectorXd& aCoordinates) const;
+
+    /// @brief                  Produce a State with the CoordinatesSubsets specified by the StateBuilder.
+    ///
+    /// @param                  [in] aState the state from which the coordinates will be taken.
+    /// @return                 A State with the CoordinatesSubsets of the StateBuilder.
+
+    const State reduce(const State& aState) const;
+
+    /// @brief                  Produce a State with the CoordinatesSubsets specified by the StateBuilder.
+    ///
+    /// @param                  [in] aState the state from which the coordinates will be taken.
```

Comment:
```suggestion
    /// @param                  [in] aState the State to be expanded
```

nit: a little clearer IMO